draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2028,1226 +1972,1147 @@ uCDN-A                         dCDN-B                         dCDN-C
-                those extensions MUST be exactly as they appear in the request.
-   <vspace blankLines="1"/>
-   Value: Array of GenericTriggerExtension objects, where each extension object is copied
-                from the "extensions" array values in the Trigger Specification.
-   <vspace blankLines="1"/>
-   Mandatory: No. The "extensions" array SHOULD be used only if the error relates to
-                extension objects. Property omission should be interpreted as 
-                "the error is not related to any extension".
-   </t>
 
-   </list></t>
-   </list></t>
+        <t>The following is an example of a JSON-serialized generic trigger extension object containing a location policy
+           object that allows the trigger execution in the US but blocks its execution in Canada:
+        </t>
+<sourcecode type="json" name="trigger-extension-json"><![CDATA[
+{
+  "generic-trigger-extension-type": "location-policy",
+  "generic-trigger-extension-value": {
+    "locations": [
+      {
+        "action": "allow",
+        "footprints": [{
+          "footprint-type": "countrycode",
+          "footprint-value": [ "us" ]
+        }]
+      },
+      {
+        "action": "deny",
+        "footprints": [{
+          "footprint-type": "countrycode",
+          "footprint-value": [ "ca" ]
+        }]
+      }
+    ]
+  },
+  "mandatory-to-enforce": true,
+  "safe-to-redistribute": true,
+  "incomprehensible": false
+}
+       ]]></sourcecode>
+      </section>
 
-   <t><list style="empty" hangIndent="3">
-   <t><list style="hanging" hangIndent="3"><t hangText="Name: cdn-id">
-   <vspace blankLines="1"/>
-   Description: The CDN PID of the CDN where the error occurred. The "cdn-id" property is used
-                by the originating uCDN or by the propagating dCDN in order to distinguish in which CDN the 
-                error occurred.
-   <vspace blankLines="1"/>
-   Value:  A non-empty JSON string, where the string is a CDN PID as defined in
-                 <xref target="loop-detection-and-prevention"/>
-   <vspace blankLines="1"/>
-   Mandatory: Yes. dCDN may use its own CDN PID if it does not want to expose the CDN PIDs of dCDNs.
-   </t>
+      <section anchor="time-policy" title="TimePolicy Extension">
+        <t>uCDN may wish to perform content management operations on dCDN on a specific schedule.
+           The TimePolicy extension allows uCDN to instruct dCDN to execute the trigger command
+           in a desired time window. For example, a content provider that wishes to pre-populate a new
+           episode at off-peak time so that it would be ready on caches at prime time when the episode
+           is released for viewing. A scheduled operation enables uCDN to direct dCDN in what
+           time frame to execute the trigger.
+        </t>
+        <t>
+           uCDN may wish to schedule a trigger such that dCDN will execute it in local time,
```

Comment:
We need to remove this paragraph--it pertains to the Local Time Window which I removed earlier (and apparently didn't completely remove).